common/transport/mqtt/test/_mqtt_base_test.js

```diff
@@ -1,89 +1,112 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-'use strict';
+"use strict";
 
-var EventEmitter = require('events').EventEmitter;
-var assert = require('chai').assert;
-var sinon = require('sinon');
+var EventEmitter = require("events").EventEmitter;
+var assert = require("chai").assert;
+var should = require("chai").should;
+var sinon = require("sinon");
 
-var MqttBase = require('../dist/mqtt_base.js').MqttBase;
-var FakeMqtt = require('./_fake_mqtt.js').FakeMqtt;
-var PubFakeMqtt = require('./_fake_mqtt.js').PubFakeMqtt;
-var PubACKTwiceFakeMqtt = require('./_fake_mqtt.js').PubACKTwiceFakeMqtt;
-var ErrorFakeMqtt = require('./_fake_mqtt.js').ErrorFakeMqtt;
-var errors = require('azure-iot-common').errors;
+var MqttBase = require("../dist/mqtt_base.js").MqttBase;
```

Comment:
curious about the change to double quotes. Is that our style standard?

Reply:
Whoops. I think my formatter did that when I saved.